soccer/gameplay/tactics/wall.py

```diff
@@ -19,12 +19,12 @@ class State(enum.Enum):
     def __init__(
             self,
             num_defenders=3,  # number of defenders we're making the wall with (default 3)
-            curvature=.3,  # 'curvature' (in radians) of the wall 
+            curvature=0,  # 'curvature' (in radians) of the wall 
```

Comment:
Why did you change these values for this commit?

Reply:
youre right this is irrelevant to this PR i'll remove the changess